src/main/webapp/assets/js/feedScript.js

```diff
@@ -785,21 +838,44 @@ function resetMarks(formElements) {
 function markInvalidInputs(invalidIds, errorMessages, formElements) {
   resetMarks(formElements);
   invalidIds.forEach((id) => {
-    const elt = formElements.namedItem(id);
+    const elt = document.getElementById(id);
     if (elt) {
-      elt.style.background = '#ff999966';
+      elt.style.backgroundColor = '#ff999966';
     }
   });
   if (errorMessages.length > 0) {
     const modalError = document.createElement('div');
     modalError.setAttribute('id', 'modal-input-error');
     modalError.setAttribute('tabindex', '0');
-    modalError.innerHTML += '<ul>';
+    let errorHTMLString = '<ul>';
     errorMessages.forEach((message) => {
-      modalError.innerHTML += '<li>' + message + '</li>';
+      errorHTMLString += '<li>' + message + '</li>';
     });
-    modalError.innerHTML += '</ul>';
-    modalForm.insertBefore(modalError, submitModalButton);
+    errorHTMLString += '</ul>';
+    modalError.innerHTML = errorHTMLString;
+    modalButtonDiv.insertBefore(modalError, submitModalButton);
+  }
+}
+
+/**
+ * Adds an error message if the file is larger than 4MB or is not an image.
+ * @param {array} invalidIds
+ * @param {array} errorMessages
+ * @return {void}
+ */
+function validateModalFile(invalidIds, errorMessages) {
+  const file = modalFileUpload.files[0];
+  if (file) {
+    if (file.type.includes('image')) {
+      const largestFileSize = 4000000;
```

Comment:
Add some comment here to clarify what this is.

Reply:
Good point!